### PR TITLE
Add JVM flag to redirect JVM crash logs to var/log

### PR DESCRIPTION
**Pull request by BrunoPSR**

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/sls-packaging](https://github.com/palantir/sls-packaging), @BrunoPSR! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

**Review by markelliot — APPROVED** [review]

**Review by iamdanfox — DISMISSED** [review]

File: gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy [hunk 1]
```diff
@@ -39,6 +39,7 @@ class LaunchConfigTask extends DefaultTask {
             '-XX:+UseGCLogFileRotation',
             '-XX:GCLogFileSize=10M',
             '-XX:NumberOfGCLogFiles=10',
+            '-XX:ErrorFile=var/log/hs_err_pid%p.log',
```
Comment (iamdanfox) [inline gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy]: Actually not sure this really belongs in the `gcJvmOpts` list (we concatenate tmpdirJvmOpts + gcJvmOpts + dnsJvmOpts + defaultJvmOpts) below.  Maybe worth a new list like `crashJvmOpts`, `loggingJvmOpts` or `debuggingJvmOpts`?

Comment (iamdanfox) [reply gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy]: Also looking at the master list of options: https://www.oracle.com/technetwork/articles/java/vmoptions-jsp-140102.html, perhaps we should make the heap dumps not land in the SLS root either (`-XX:HeapDumpPath=./java_pid<pid>.hprof`)?

Comment (BrunoPSR) [reply gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy]: Yeah was wondering that myself and wasn't sure if I should put it in a dedicated variable. I moved it to loggingJvmOpts and amended the commit. Will make a separate commit for the heap dumps.

Comment (BrunoPSR) [reply gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy]: Actually, did you mean `-XX:HeapDumpPath=var/log/java_pid%p.hprof`?

**Review by iamdanfox — APPROVED** [review]

**Pull request merged**